src/main/java/com/programmers/springweekly/util/Validator.java

```diff
@@ -0,0 +1,40 @@
+package com.programmers.springweekly.util;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class Validator {
```

Comment:
기본 생성자 부분도 lombok을 활용할 수 있습니다

```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class Validator {
```